src/tcgwars/logic/impl/gen8/BattleStyles.groovy

```diff
@@ -3000,24 +3000,24 @@ public enum BattleStyles implements LogicCardInfo {
       case PHOEBE_130:
       return supporter (this) {
         text "During this turn, damage from your Pokémon VMAX's attacks isn't affected by any effects on your opponent's Active Pokémon."
-        delayed {
-          //TODO Fix, not working currently against safeguard (Keldeo-GX) (copy of Ultra Forest Kartenvoy)
-          before PROCESS_ATTACK_EFFECTS, {
-            if (ef.attacker.topPokemonCard.cardTypes.is(VMAX)) {
-              bg.dm().each {
-                if (it.to.owner != self.owner && it.to.active) {
-                  bc "Phoebe's effect activated"
-                  it.flags.add(DamageManager.DamageFlag.NO_DEFENDING_EFFECT)
+        onPlay {
+          delayed {
+            //TODO Fix, not working currently against safeguard (Keldeo-GX) (copy of Ultra Forest Kartenvoy)
+            before PROCESS_ATTACK_EFFECTS, {
+              if (ef.attacker.topPokemonCard.cardTypes.is(VMAX)) {
```

Comment:
Dont' use `cardTypes` for PokemonCardSets. This one is missing regular Pokemon V since it isn't using the helper method on PokemonCardSet